docs/api-guide/serializers.md

```diff
@@ -189,6 +189,12 @@ Your `validate_<field_name>` methods should return the validated value or raise
                 raise serializers.ValidationError("Blog post is not about Django")
             return value
 
+---
+
+**Note:** If your `<field_name>` is declared on your model with the parameter `required=False` then this validation step will not take place.
```

Comment:
Shouldn't the wording be "will not take place if the field is not included"? The validation should be triggered when a value is passed in, just it will be skipped if there is nothing to validate.